File: lua/src/lang_test/control_test.lua
-- -*- coding:utf-8 -*-
-- author:liushangliang
-- desc: 控制语句测试

local lu = require("luaunit")

_G.TestIO = {}

function _G.TestControl.IF()
    local var = false
    if var == 1  then
        print(1)
    elseif var == 2 then
        print(2)
    else
        print("else")
    end


end

os.exit(lu.LuaUnit.run())

File: python/src/lang_test/control_test.py
# -*- coding:utf-8; -*-
# author: liushangliang
# desc: 控制语句测试

import unittest


class ControlTest(unittest.TestCase):
    def testIF(self):
        var = 1
        if var == 1:
            print(1)
        elif var == 2:
            print(2)
        else:
            print(3)


if __name__ == "__main__":
    unittest.main()
